.eslintrc.js

```diff
@@ -61,6 +61,8 @@ module.exports = {
     'mocha/no-top-level-hooks': 'error',
     'mocha/valid-suite-description': 'error',
 
+    // This rule is great for raising people awareness of what a key is and how it works.
+    'react/no-array-index-key': 'off',
```

Comment:
:tada:

Reply:
It's the beginning of the dismantling of #15703 😆